src/main/java/org/cascadebot/cascadebot/commands/developer/EvalCommand.java

```diff
@@ -108,7 +108,7 @@ public String command() {
 
     @Override
     public String description() {
-        return "evaluate code";
+        return "Evaluate code";
```

Comment:
Same

Reply:
grammar tho

Reply:
Change it to
```suggestion
        return "Evaluates code";
``` 
then ;)